core/mocks/init.ts

```diff
@@ -0,0 +1,10 @@
+export default () => {
+  if (
+    process.env.NEXT_PUBLIC_API_MOCKING === 'enabled' &&
+    process.env.NODE_ENV !== 'production'
+  ) {
+    // eslint-disable-next-line global-require
+    const { startMSW } = require('.');
+    startMSW();
```

Comment:
We're doing conditional require here to avoid msw being added to production bundle